src/pages/admin/RoomForm.js

```diff
@@ -5,24 +5,26 @@ import Alert from '@mui/material/Alert';
 import { useFormik } from 'formik';
 import { TextField, Select, MenuItem, FormControl, InputLabel, Grid } from '@mui/material';
 import useCreate12CandlesRoom from '../../hooks/useCreate12CandlesRoom';
-
-const initialValues = {
-  name: '',
-  description: '',
-  maxPlayers: '',
-  minPlayers: '',
-  maxOrders: '',
-  playerCapital: '',
-  startDateTime: '',
-  endDateTime: '',
-  marketType: ''
-};
-
-const CreateRoomForm = () => {
+import useEdit12CandlesRooms from 'hooks/useEdit12CandlesRooms';
+const CreateRoomForm = ({ item, isViewable, isEditable }) => {
   const { mutateAsync } = useCreate12CandlesRoom();
+  const { mutateAsync: mutateAsync1 } = useEdit12CandlesRooms();
```

Comment:
Please rename the variables as below

```suggestion
  const { mutateAsync: mutateCreateRoomAsync } = useCreate12CandlesRoom();
  const { mutateAsync: mutateEditRoomAsync } = useEdit12CandlesRooms();
```

Reply:
done